test/dummy/app/models/binary_store_record.rb

```diff
@@ -2,4 +2,5 @@ class BinaryStoreRecord < ApplicationRecord
   include StructuredStore::Storable
 
   store :store, coder: JSON
+  structured_store :store
```

Comment:
@timgentry  Should it be other way around ? Defining the store column first followed by its coder ?
```
 structured_store :store
 store :store, coder: JSON
```